Error building on Linux: electron-v1.7-linux-x64/node_sqlite3.node'

Operation System: Linux (Debian 8.7, x86_64)
Beaker Version: 0.7.2

Related: #267 and #224 

```
$ npm start

> @ start /home/user/Downloads/beaker-0.7.2
> gulp start

[02:28:03] Using gulpfile ~/Downloads/beaker-0.7.2/gulpfile.js
[02:28:03] Starting 'start'...
Spawning electron /home/user/Downloads/beaker-0.7.2/node_modules/electron/dist/electron
[02:28:03] Finished 'start' after 22 ms
App threw an error during load
Error: Cannot find module '/home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3/lib/binding/electron-v1.7-linux-x64/node_sqlite3.node'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (/home/user/Downloads/beaker-0.7.2/node_modules/electron/dist/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3/lib/sqlite3.js:4:15)
    at Object.<anonymous> (/home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3/lib/sqlite3.js:190:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
A JavaScript error occurred in the main process
Uncaught Exception:
Error: Cannot find module '/home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3/lib/binding/electron-v1.7-linux-x64/node_sqlite3.node'
    at Module._resolveFilename (module.js:470:15)
    at Function.Module._resolveFilename (/home/user/Downloads/beaker-0.7.2/node_modules/electron/dist/resources/electron.asar/common/reset-search-paths.js:35:12)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (/home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3/lib/sqlite3.js:4:15)
    at Object.<anonymous> (/home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3/lib/sqlite3.js:190:3)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
```

The sqlite issue is odd considering that `npm install` explicitly said that sqlite was installed:

```
beaker-0.7.2$ npm install
npm WARN deprecated minimatch@2.0.10: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated minimatch@0.2.14: Please update to minimatch 3.0.2 or higher to avoid a RegExp DoS issue
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v7.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.

> electron@1.7.1 postinstall /home/user/Downloads/beaker-0.7.2/node_modules/electron
> node install.js


> undefined postinstall /home/user/Downloads/beaker-0.7.2
> cd app && npm install

npm WARN deprecated fs-promise@2.0.3: Use mz or fs-extra^3.0 with Promise Support

> sodium-native@1.10.1 install /home/user/Downloads/beaker-0.7.2/app/node_modules/sodium-native
> node-gyp-build "node preinstall.js" "node postinstall.js"


> utp-native@1.5.1 install /home/user/Downloads/beaker-0.7.2/app/node_modules/utp-native
> node-gyp-build


> sqlite3@3.1.8 install /home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3
> node-pre-gyp install --fallback-to-build

[sqlite3] Success: "/home/user/Downloads/beaker-0.7.2/app/node_modules/sqlite3/lib/binding/node-v51-linux-x64/node_sqlite3.node" is installed via remote
```

--- a/README.md
+++ b/README.md
@@ -96,10 +96,10 @@ console.log(archive.url)
 ## Building from source
 
 Requires node 6.2.1.
-In linux (possibly also OSX) you need libtool, m4, and automake.
+In linux (possibly also OSX) you need libtool, m4, automake, make, and g++.
 
 ```
-sudo apt-get install libtool m4 automake
+sudo apt-get install libtool m4 automake make g++
 ```
 
 To build:

